src/Symfony/Component/Notifier/Bridge/Telegram/README.md

```diff
@@ -14,6 +14,25 @@ where:
  - `TOKEN` is your Telegram token
  - `CHAT_ID` is your Telegram chat id
 
+Interacting with local API server instead of official Telegram API
+------------------------------------------------------------------
+
+If such a case is needed, you can simply replace the `default` keyword in the DSN
+with the desired domain/IP address of your local API server. You may also want to
+disable the notifier default behavior of using `https` protocol as local API servers
+can only accept `http` traffic.
+
+DSN Example:
+```
+TELEGRAM_DSN=telegram://TOKEN@localhost:5001?channel=CHAT_ID&disable_https=1
+```
+
+Caution: Disabling the use of the `https` protocol can pose a security risk.
+You should only do this if your local API server is hosted somehow internally
+and the traffic will flow over a secure environment. Otherwise,
+you may want to put a TLS-termination server in front of your local API server
+to use it normally over `https` protocol.
+
```

Comment:
```suggestion
If such a case is needed, you can replace the `default` keyword in the DSN
with the desired domain/IP address of your local API server. You may also want to
disable the notifier default behavior of using `https` protocol as local API servers
can only accept `http` traffic.
Example:
```
TELEGRAM_DSN=telegram://TOKEN@localhost:5001?channel=CHAT_ID&disable_https=1
```

Caution: Disabling the use of the `https` protocol can pose a security risk.
You should only do this if your local API server is hosted somehow internally
and the traffic will flow over a secure environment. Otherwise,
you may want to put a TLS-termination server in front of your local API server
to use it normally over `https` protocol.

```